internal/commands/pull/command_test.go

```diff
@@ -100,7 +100,6 @@ Contents would have been written to: app
 			profile := mock.NewProfile(t)
 
 			out, ui := mock.NewUI()
-
```

Comment:
probably not a necessary diff here

Reply:
for some reason, adding this space caused the linter to fail: https://spruce.mongodb.com/task/realm_cli_rhel70_lint_patch_0a4628b5d4bc5ae8593504528accb8e1a22369d4_610858c51e2d171708d1876a_21_08_02_20_42_46/logs?execution=0

so i just removed it